src/background/services/openPayments.ts

```diff
@@ -1023,6 +1070,15 @@ export const isOutOfBalanceError = (error: any) => {
   return error.status === 403 && error.description === 'unauthorized';
 };
 
+export const isMissingGrantPermissionsError = (error: any) => {
+  if (!isOpenPaymentsClientError(error)) return false;
+  return (
+    error.status === 403 &&
+    (error.description === 'Insufficient Grant' /* Fynbos */ ||
+      error.description === 'Inactive Token')
```

Comment:
We shouldn't check for `Inactive Token` here. We are checking for this description with `isTokenExpiredError`
```suggestion
      error.description === 'Inactive Token')
```